### DISCUSSION TITLE
How to use DBS, BCSE and other variance reduction techniques in egs++?

### DISCUSSION THREAD

**Discussion by CQUswl** [nrc-cnrc/EGSnrc#1122] [General]
I'd like to make full use of the variance reduction techniques in the egs++ code, but I haven't found instructions on how to use DBS, BCSE, etc.

**Comment by CQUswl** [comment]
> I'd like to make full use of the variance reduction techniques in the egs++ code, but I haven't found instructions on how to use DBS, BCSE, etc.

I don't know if these variance reduction tricks from BEAMnrc are available in egs++.

**Reply by rtownson** [reply]
The `egs_chamber` application has a variety of VRTs built in, I would start there. Uniform brems splitting can be done using:

https://nrc-cnrc.github.io/EGSnrc/doc/pirs898/classEGS__RadiativeSplitting.html

We are currently working on DBS (https://github.com/nrc-cnrc/EGSnrc/pull/974) but it's not ready for use yet.

**Reply by CQUswl** [reply]
Thank you very much for your help. I see in egs_radiative_splitting.h that in addition to URS, there are DRS and DRSf, is there any difference between these three?

**Reply by rtownson** [reply]
The references to DRS and DRSf are just placeholders, they are not used anywhere yet.

Note that radiative splitting recently had a bug fix: https://github.com/nrc-cnrc/EGSnrc/pull/1011

If the radiative splitting is needed, you may want to checkout that PR to get the changes.
